src/compositions/question-form-8.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC, useEffect, useState } from 'react';
+import { InputField } from '../elements/input-field';
+import { useRouter } from 'next/dist/client/router';
+import { Copy } from '../identity/copy';
+
+type Props = {
+  label?: string;
+  href: string;
+  answer: { [key: string]: string };
+  callback: React.Dispatch<React.SetStateAction<{ [key: string]: string }>>;
+};
+
+const validateInput = (answer: string) => {
+  if (parseFloat(answer) > 100) {
+    return '100 prozent ist das max';
+  }
+  if (parseFloat(answer) < 0) {
+    return 'Die Prozentangabe darf nicht negativ sein.';
+  }
+  return null;
+};
+export const Question8: FC<Props> = ({ label, href, callback, answer }) => {
+  const initialAnswer = '25%';
+  const router = useRouter();
+  const [totalAmount, setTotalAmount] = useState(0);
+
+  useEffect(() => {
+    if (answer === { Auto: '', ÖV: '', Fahrrad: '', Fuss: '' }) {
```

Comment:
Soooo back to the value / text label thingy. This is exactly why you want to separate them :D